interfaces/builtin/modem_manager.go

```diff
@@ -47,6 +47,10 @@ var modemManagerPermanentSlotAppArmor = []byte(`
 # For ioctl TIOCSSERIAL ASYNC_CLOSING_WAIT_NONE
 capability sys_admin,
 
+# For {mbim,qmi}-proxy
+unix (create,bind,listen,send,receive) type=stream,
```

Comment:
These are the denials I see:

[62495.225343] audit: type=1400 audit(1484295707.162:99): apparmor="DENIED" operation="bind" profile="snap.modem-manager.modemmanager" pid=5080 comm="mbim-proxy" family="unix" sock_type="stream" protocol=0 requested_mask="bind" denied_mask="bind" addr="@mbim-proxy"

After letting it bind:

[63467.799248] audit: type=1400 audit(1484296678.748:132): apparmor="DENIED" operation="listen" profile="snap.modem-manager.modemmanager" pid=5781 comm="mbim-proxy" family="unix" sock_type="stream" protocol=0 requested_mask="listen" denied_mask="listen" addr="@mbim-proxy"

It looks like (bind,listen,send,receive) is enough, create can be dropped. There is no label in the denials, but I guess "profile" can be used for matching.

The denials come from the mbim-proxy process, which is created by ModemManager to handle mbim type modems.